homeassistant/components/harmony/translations/nl.json

```diff
@@ -27,7 +27,8 @@
             "init": {
                 "data": {
                     "activity": "De standaardactiviteit die moet worden uitgevoerd wanneer er geen is opgegeven.",
-                    "delay_secs": "De vertraging tussen het verzenden van opdrachten."
+                    "delay_secs": "De vertraging tussen het verzenden van opdrachten.",
+                    "activity_notify": "Verander de activiteit bij het starten van de activiteits verandering."
```

Comment:
No need to add this here as it will get overwritten by lokalise.  Only `strings.json` and `en.json` should be in the PR

Reply:
And here I was trying to be helpful. :-) 
Reverted.